dockstore-client/src/main/java/io/dockstore/client/cli/nested/WesLauncher.java

```diff
@@ -233,13 +233,13 @@ public static String combineTrsUrlComponents(WorkflowClient workflowClient, Stri
             return Optional.empty();
         }
 
-        // Verify the file path exists
-        // 1. If the path is to a directory, return an empty optional
+        // If the path isn't a file or a directory, we've done something wrong on our end.
+        // We don't want to attach a directory directly, so return an empty optional instead
         Path path = Paths.get(filePath);
         if (Files.isDirectory(path)) {
             return Optional.empty();
         } else if (!Files.isRegularFile(path)) {
```

Comment:
Will this fail on symbolic links? If so, do we want that?

If we're talking about uploading genomic data (are we?), then I could see how you might use them.

If I'm right, can be a follow up ticket.

Reply:
Good point, I hadn't considered that. 

Taking a look at the Java Docs (https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Files.html#isRegularFile(java.nio.file.Path,java.nio.file.LinkOption...), symbolic links are followed by default, and should be read properly:

> By default, symbolic links are followed and the file attribute of the final target of the link is read.